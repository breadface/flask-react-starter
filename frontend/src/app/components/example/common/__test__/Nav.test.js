import React from 'react'
import Nav from './../Nav'
import renderer from 'react-test-renderer'

describe('Nav component', () => {
  it('renders without errors', () => {
    const tree = renderer.create(<Nav />)
    .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
