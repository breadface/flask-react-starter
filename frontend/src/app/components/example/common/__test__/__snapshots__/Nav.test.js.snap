exports[`NAV component renders without errors 1`] = `
<div>
  <a
    onClick={[Function]}
    style={Object {}}>
    <h1>
      Flask React Starter Kit(dockerized)
    </h1>
  </a>
  <ul>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        Home
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        Blog
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        Media
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        Store
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        About
      </a>
    </li>
    <li>
      <a
        onClick={[Function]}
        style={Object {}}>
        Contact
      </a>
    </li>
  </ul>
</div>
`;
